Userland/Applications/Assistant/Providers.h

```diff
@@ -18,6 +18,8 @@
 
 namespace Assistant {
 
+static constexpr size_t MAX_SEARCH_RESULTS = 6;
```

Comment:
Can you rename this accordingly to the CodingStyle?

Reply:
I couldn't find any mention of this in `CodingStyle.md`.

After a quick grep, there seem to be a similar number of `static constexpr` variable declarations that are `UPPER_SNAKE_CASE` and `lower_snake_case`.

I'm not sure which is preferred.

Reply:
> ### Names
>A combination of CamelCase and snake_case. Use CamelCase (Capitalize the first letter, including all letters in an acronym) in a class, struct, or namespace name. Use snake_case (all lowercase, with underscores separating words) for variable and function names.

Reply:
Ah, that bit right at the top! Kind of obvious now you point it out :laughing:. 

I've changed it to `snake_case`.

Reply:
@LucasChollet that's outdated, the preferred spelling for constants is ALL_CAPS.
@tcl3 please revert the change :)

Reply:
noooooooooooooooooooo

Reply:
Reverted to follow the updated `CodingStyle.md`.